pkg/comment_actions/parsing.go

```diff
@@ -29,15 +29,34 @@ type CommentArgs struct {
 	Rest    []string
 }
 
+func Contains(s []string, str string) bool {
+	existedOnce := false
+	for _, v := range s {
+		if v == str {
+			existedOnce = true
+		}
+	}
+
+	return existedOnce
+}
+
 func ParseCommentCommand(noteBody string) (*CommentOpts, error) {
 	comment := strings.TrimSpace(strings.ToLower(noteBody))
 	words := strings.Fields(comment)
+	escapeKeyword := []string{"-e", "--allow_empty_run"}
 
 	if len(words) == 0 {
 		return nil, ErrNoNotePassed
 	}
 
-	if len(words)%2 != 0 {
+	isEscaped := false
+	for _, a := range escapeKeyword {
```

Comment:
We can probably drop this and replace it with something like:
```
if len(words)%2 != 0 && (slices.Contains(words, "-e") || slices.Contains(words, "--allow_empty_run")) { ... }
``` 

As of Go 1.21 this is now part of the standard library. Example: https://go.dev/play/p/cxVe9DRKfXQ

Reply:
I think we can just drop this now since we check if there's any commands and then we also validate the flags https://github.com/zapier/tfbuddy/blob/main/pkg/comment_actions/parsing.go#L49. We should also update the unit test for this function to ensure that it works as expected wth valid and invalid flags passed.

Reply:
will remove this checking